docker/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     volumes:
       - ${WARDEN_HOME_DIR}/etc/traefik/traefik.yml:/etc/traefik/traefik.yml
       - ${WARDEN_HOME_DIR}/etc/traefik/dynamic.yml:/etc/traefik/dynamic.yml
-      - ${WARDEN_HOME_DIR}/ssl/certs:/etc/ssl/certs
+      - ${WARDEN_HOME_DIR}/ssl/certs:/etc/ssl/certs/warden
```

Comment:
The SSL certificate location change is outside the scope of your PR

Reply:
The original mount ends up clobbering all the default system certificates. Preventing Traefik from downloading  plugins due to SSL verification errors.
Could be split to a seperate PR, as it should affecting install any Traefik plugin

Reply:
@SamJUK To @navarr 's point, then there should be a separate PR that fixes the clobbering issue, and this one requires that one to be fixed first. The PR should do a single thing (add Sablier) not multiple (add Sablier, change SSL dirs, etc.)

Reply:
Fair points, separated this change to PR #812